packages/react-scripts/backpack-addons/cssModules.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+const postcssNormalize = require('postcss-normalize');
+const paths = require('../config/paths');
+const appPackageJson = require(paths.appPackageJson);
+const bpkReactScriptsConfig = appPackageJson['backpack-react-scripts'] || {};
+const cssModulesEnabled = bpkReactScriptsConfig.cssModules !== false;
+
+// Backpack / saddlebag node module regexes
+const backpackModulesRegex = /node_modules[\\/]bpk-/;
+const scopedBackpackModulesRegex = /node_modules[\\/]@skyscanner[\\/]bpk-/;
+
+const getStyleTestRegexes = regexType => {
+  // style files regexes, the regex values should keep up to date with webpack.config.js
+  const cssRegex = /\.css$/;
+  const cssModuleRegex = /\.module\.css$/;
+  const sassRegex = /\.(scss|sass)$/;
+  const sassModuleRegex = /\.module\.(scss|sass)$/;
+
+  switch (regexType) {
+    case 'css':
+      return {
+        and: [cssRegex, () => !cssModulesEnabled],
+        not: [backpackModulesRegex, scopedBackpackModulesRegex],
```

Comment:
Accroding webpack5 [Rule.exclude](https://webpack.js.org/configuration/module/#ruleexclude) and Rule.include:  If you supply a Rule.exclude option, you cannot also supply a Rule.resource(and, not, or )
So I'm going to change `exclude` here to `not`